docs/Collect Submissions/3-Build a Form/sections.md

```diff
@@ -27,8 +27,8 @@ Form sections are used to group related questions together into categories. Sect
 
 1. On the right hand corner of a section, click **⋮**.
 2. Click **Move section**. A popup should appear.
-3. To re-order sections, drag the sections in the order you'd like.
-4. Click **Save**.
+3. To re-order sections, use the **˄** and **˅** buttons to move sections in the order you'd like.
```

Comment:
We've disabled drag and drop since it was broken on this page. It looks like this now and we use the arrows to move the sections around.
![image](https://user-images.githubusercontent.com/13268990/154317269-5dc9176a-e52e-4f81-8564-d663bd2a0535.png)